### DISCUSSION TITLE
Red shadows and green darkness bug?

### DISCUSSION THREAD

**Discussion by GreenGroove46** [sultim-t/xash-rt#189] [Q&A]
I don't know why, but i have red sun shadows, and green dark, (My videocard its NVIDIA Geforce RTX 2060)
![20230421191456_1](https://user-images.githubusercontent.com/126620497/233685938-9b88f523-d06c-4b14-b7fa-4038c7574ff4.jpg)

**Comment by brurpo** [comment]
same issue, 2080ti
any ideas?

**Reply by TuTuc0** [reply]
3070, same issue. Could be new driver update?

**Comment by GreenGroove46** [comment]
If you are playing with the nvidia studio driver, try to download and install game ready driver. I fixed it with this method.

**Comment by Gibshooter** [comment]
Hey there. I had the same issue with an RTX 3060 Ti today, with and without DLSS.

I installed the latest game ready driver as well + system restart, but that did not help.

![Zwischenablage01](https://github.com/sultim-t/xash-rt/assets/23134973/860b523d-d80d-4421-b105-98e0c0e02e8b)

**Comment by GreenGroove46** [comment]
Guys, i don't know how to fix this bug.

**Comment by sultim-t** [comment]
Potential fix: https://github.com/sultim-t/xash-rt/issues/191#issuecomment-1602481771
